SPECS/libndp/libndp.spec

```diff
@@ -1,23 +1,23 @@
 Summary:        Library for Neighbor Discovery Protocol
 Name:           libndp
-Version:        1.7
-Release:        3%{?dist}
+Version:        1.8
+Release:        1%{?dist}
 License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
 URL:            http://www.libndp.org/
 Source:         http://www.libndp.org/files/%{name}-%{version}.tar.gz
 %define sha1 libndp=1458d2a70c6bc4cdcbf525e02582060e799778bc
```

Comment:
Could you also remove the `sha1` macro while you're here?

Reply:
Fixed